source/common/http/conn_manager_utility.cc

```diff
@@ -223,12 +229,17 @@ void ConnectionManagerUtility::mutateXfccRequestHeader(Http::HeaderMap& request_
         // The "SAN" key still exists even if the SAN is empty.
         client_cert_details.push_back("SAN=" + connection.ssl()->uriSanPeerCertificate());
         break;
-      case Http::ClientCertDetailsType::Cert:
-        const std::string peer_cert = connection.ssl()->urlEncodedPemEncodedPeerCertificate();
-        if (!peer_cert.empty()) {
-          client_cert_details.push_back("Cert=\"" + peer_cert + "\"");
-        }
+      case Http::ClientCertDetailsType::URI:
+        client_cert_details.push_back("URI=" + connection.ssl()->uriSanPeerCertificate());
```

Comment:
Is this guaranteed to exist? Why don't we check it like the others?

Reply:
SAN has a comment `// The "SAN" key still exists even if the SAN is empty.`
As this is replacing SAN, kept the behavior for URI consistent with SAN.

Reply:
Also, it's not guaranteed to exist. In that case, XFCC will have an empty value key, URI=

Reply:
OK SGTM